NethermindEth/dotnet-libp2p
# Pubsub discovery

- Add [pubsub peer discovery](https://github.com/libp2p/js-libp2p-pubsub-peer-discovery)
- Better support for gossipsub [peer exchange](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#prune-backoff-and-peer-exchange)
- Add peer store prototype

Testing:
- Add new test muxer that allows to spawn peers without relying on real network connectivity
- Add tests to confirm pubsub peer discovery works and messages propagate
